LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -12,53 +12,82 @@ import LoopKit
 import SwiftUI
 
 public struct TherapySettingsView: View, HorizontalSizeClassOverride {
+    public struct ActionButton {
+        public init(localizedString: String, action: @escaping () -> Void) {
+            self.localizedString = localizedString
+            self.action = action
+        }
+        let localizedString: String
+        let action: () -> Void
+    }
+    
     @Environment(\.dismiss) var dismiss
         
     @ObservedObject var viewModel: TherapySettingsViewModel
     
     @State var isEditing: Bool = false
     
-    private let mode: PresentationMode
```

Comment:
Moved `mode` into `TherapySettingsViewModel` because the ViewModel is what is shared amongst the screens in the two "incarnations" of editing/reviewing TherapySettings.  Seemed to make sense to me.